diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/SingleLineStatementsRuleWarnTest.kt

```diff
@@ -0,0 +1,140 @@
+package org.cqfn.diktat.ruleset.chapter3
+
+import com.pinterest.ktlint.core.LintError
+import org.cqfn.diktat.ruleset.constants.Warnings.MORE_THAN_ONE_STATEMENT_PER_LINE
+import org.cqfn.diktat.ruleset.rules.DIKTAT_RULE_SET_ID
+import org.cqfn.diktat.ruleset.rules.SingleLineStatementsRule
+import org.cqfn.diktat.util.lintMethod
+import org.junit.Test
+
+class SingleLineStatementsRuleWarnTest {
+
+    private val ruleId = "$DIKTAT_RULE_SET_ID:statement"
+
+
+    @Test
+    fun `check two statement per line`() {
+        lintMethod(SingleLineStatementsRule(),
+                """
+                    |import com.pinterest.ktlint.core.KtLint; import com.pinterest.ktlint.core.LintError
+                    |
+                    |fun foo() {
+                    |    if (x < -5) {
+                    |       goo(); hoo()
+                    |    }
+                    |    else {
+                    |    }
+                    |    
+                    |    when(x) {
+                    |       1 -> println(1)
+                    |       else -> println("3;5")
+                    |    }
+                    |    val a = 5; val b = 10
+                    |    println(1); println(1)
+                    |}
+                """.trimMargin(),
+                LintError(1,40,ruleId,"${MORE_THAN_ONE_STATEMENT_PER_LINE.warnText()} import com.pinterest.ktlint.core.KtLint; import com.pinterest.ktlint.core.LintError", false),
```

Comment:
```suggestion
                LintError(1,40,ruleId,"${MORE_THAN_ONE_STATEMENT_PER_LINE.warnText()} import com.pinterest.ktlint.core.KtLint; import com.pinterest.ktlint.core.LintError", true),
```
Why `false` here and everywhere else? This test passes because this field isn't checked (well, this is our fault due to some historical reasons, there is an open issue to fix this: #130), but you warning has `canBeAutoCorrected` set to `true`!